src/swagger.cr

```diff
@@ -4,9 +4,6 @@ require "./swagger/*"
 module Swagger
   VERSION = "0.1.1"
```

Comment:
BTW, could be removed too - don't see how it is useful?
Furthermore, for each release, we have to think about:
- git tag
- shard.yml version
- changing this one :confused:

Reply:
For now, i have no idea. we can remove it. 🤣